### PR TITLE
add-filtering-capabilities-to-dataspace-file-browsers

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

File: app/resources/locales/locale-fr.json [hunk 1]
```diff
@@ -124,5 +124,7 @@
   "Yes, download it!": "Oui, téléchargez-le!",
   "Yes, stop uploading!": "Oui, arrêtez de télécharger!",
   "You are about to download the folder": "Vous êtes sur le point de télécharger le dossier",
-  "Your": "Votre"
+  "Your": "Votre",
+  "Filter:": "Filtre:",
+  "Filter files according to a glob pattern, e.g. *.xml, image*": "Filtrer les fichiers selon le glob modèle, par ex. *.xml, image*"
```
Comment (fviale) [inline app/resources/locales/locale-fr.json:129]: Filtrer les fichiers selon une glob pattern, par ex. *.xml, image*

Sometimes technical words cannot be translated

Comment (fviale) [reply app/resources/locales/locale-fr.json:129]: Same remark for other languages (spanish, protuguese)

**Review by fviale — APPROVED** [review]
minor remark regarding translation

**Review by mboussaa — APPROVED** [review]

File: app/resources/locales/locale-fr.json (hunk 1 above)
Comment (oanaBiancaSchiopu) [reply app/resources/locales/locale-fr.json:129]: thanks @fviale I will modify

**Pull request merged**